language-support/hs/bindings/src/DA/Ledger/Convert.hs

```diff
@@ -79,7 +79,26 @@ lowerCommands = \case
         commandsDeduplicationPeriod = fmap lowerDeduplicationPeriod dedupPeriod,
         commandsCommands = Vector.fromList $ map lowerCommand coms,
         commandsMinLedgerTimeAbs = fmap lowerTimestamp minLeTimeAbs,
-        commandsMinLedgerTimeRel = minLeTimeRel }
+        commandsMinLedgerTimeRel = minLeTimeRel,
+        commandsDisclosedContracts = Vector.fromList $ map lowerDisclosedContract disContracts }
```

Comment:
given the state of the Haskell ledger API bindings (internal use only in daml-helper), we may just want to set this to an empty vector and only expose it when we need it. wdyt? cc @ray-roestenburg-da